executor/runtime/types/pod.go

```diff
@@ -475,13 +475,18 @@ func (c *PodContainer) LogUploadThresholdTime() *time.Duration {
 }
 
 func (c *PodContainer) MetatronCreds() *titus.ContainerInfo_MetatronCreds {
-	if c.podConfig.AppMetadata == nil || c.podConfig.AppMetadataSig == nil {
+	if c.titusInfo != nil {
```

Comment:
Stashed changes that I forgot to push as part of the last PR. :(

Reply:
Hmm. I kinda expected to see this code on only the cinfo type implementation, but I guess we are getting to the point where *both* need it as we make the transition?